cancelar transaçoes pendentes na external modules

gabr1gus opened the pull request:
Simplesmente deleta a transaçao pendente no external module

dependabot[bot] committed:
Bump acorn from 7.1.0 to 7.1.1 (#52)

GCastilho committed:
Implementado fee no sistema de saque (#53)

-- Resolvido um bug no script de compilação que impedia que o código fosse compilado em máquinas linux
-- Reprojetado o array 'currenciesDetailed' da currencyApi na função 'detailsOf', que retorna um objeto com code e decimals
-- Adicionado uma propriedade nas currencies para ser o fee de saque no valor de 0.001 para a nano e bitcoin
-- A função detailsOf agora também retorna o fee de saque de uma currency
-- O evento 'list' do websocket agora retorna o fee como uma das propriedades
-- Adicionado fee na aba da withdraw do site, junto com um campo mostrando o quando o usuário irá receber após o desconto do fee
-- A currencyApi agora desconta o fee de uma transação antes de salvá-la no database
-- A collection das transações agora tem um campo 'fee', que é o valor cobrado de taxa para sua execução
-- Os testes de envio de requests da CurrencyApi foram adaptados para esperarem uma ordem de saque com o fee descontado do amount
-- Refatorado e corrigidos bugs nos testes da CurrencyApi

GCastilho committed:
Configurado Continuous Integration (#56)

-- Configurado projeto para utilizar o GitHub Actions para realizar Continuous Integration no projeto, testando-o automaticamente a cada pull request

GCastilho commented:
Pq tem a label "documentation"?

E pq tem esse monte de arquivo modificado nada a ver com a external modules?

joaojvds committed:
Implementação da HTTP REST API (#51)

Refactor do webserver:
-- A pasta router agora é a pasta base do web server inteiro, não apenas do roteador
-- Adicionado um handler para subdomínios, requests para um subdomínio inexistente retornam 404
-- Requests sem um subdomínio são roteados à pasta root

HTTP API
-- Adicionado subdomínio 'api' para requests à httpApi
-- Adicionado endpoint /v1 à API
-- Implementado uma api para requests HTTP no padrão REST para substituir o websocket em requests que não precisem ser em tempo real

Geral:
-- O método getBalance da User agora pode retornar os saldos em string ou Decimal128
-- Libs que simplesmente expandiam types do typescript foram movidas para uma pasta @types, a pasta libs será apenas para módulos que expandem funções de um módulo
-- Adicionado type para permitir referência à user no request do express
-- Resolvido um bug no script de compilação que impedia a compilação em sistemas linux
-- Implementados testes para a HTTP API

Co-authored-by: GCastilho <castilho.gabriel14@gmail.com>

gabr1gus commented:
> Pq tem a label "documentation"?
> 
> E pq tem esse monte de arquivo modificado nada a ver com a external modules?

eu não faço a menor ideia, a label documentation provavelmente foi eu clicando errado, mas os arquivos modificados... não tenho ideia de o que aconteceu
